test/integration/travel_advice_test.rb

```diff
@@ -57,7 +57,7 @@ class TravelAdviceTest < ActionDispatch::IntegrationTest
     assert page.has_css?("h1", text: first_part['title'])
     assert_has_component_govspeak(first_part['body'])
 
-    refute page.has_css?(".map")
+    assert page.has_css?("figure img[src=\"#{@content_item['details']['image']['url']}\"][alt=\"#{@content_item['details']['image']['alt_text']}\"]")
```

Comment:
This should be refuting something.

Reply:
ah wondered why the test was failing thanks Paul, teaches me to copy paste 🔫